examples/FadeExample.m

```diff
@@ -89,4 +89,37 @@ - (void)startWithContext:(id<MDMTransitionContext>)context {
   [CATransaction commit];
 }
 
+
+dispatch_source_t CreateDispatchTimer(double interval, dispatch_queue_t queue, dispatch_block_t block)
+{
+  dispatch_source_t timer = dispatch_source_create(DISPATCH_SOURCE_TYPE_TIMER, 0, 0, queue);
+  if (timer)
+  {
+    dispatch_source_set_timer(timer, dispatch_time(DISPATCH_TIME_NOW, interval * NSEC_PER_SEC), interval * NSEC_PER_SEC, (1ull * NSEC_PER_SEC) / 10);
+    dispatch_source_set_event_handler(timer, block);
+    dispatch_resume(timer);
+  }
+  return timer;
+}
+
+//uncomment to make animation interactive
+/*
+- (void)startWithInteractiveContext:(nonnull id<MDMInteractiveTransitionContext>)context {
+  
+  dispatch_source_t timer = dispatch_source_create(DISPATCH_SOURCE_TYPE_TIMER, 0, 0, dispatch_get_main_queue());
+  double interval = 0.01f;
+  
+  dispatch_source_set_timer(timer, dispatch_time(DISPATCH_TIME_NOW, interval * NSEC_PER_SEC), interval * NSEC_PER_SEC, 0);
+  __block float i = 0.01;
+  dispatch_source_set_event_handler(timer, ^{
+    i += 0.01;
+    [context updatePercent: i];
+    if(i > 1.0) {
+      [context finishInteractiveTransition];
+      dispatch_source_cancel(timer);
+    }
+  });
+  dispatch_resume(timer);
+}
+ */
```

Comment:
Similar to above: let's create a new example that specifically demonstrates interactive transitioning.